docs/root/configuration/observability/access_log/stats.rst

```diff
@@ -0,0 +1,18 @@
+.. _config_access_log_stats:
+
+Statistics
+==========
+
+Currently only the gRPC access log has statistics.
```

Comment:
You should check but I think these stats: https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/statistics#file-system

Are primarily (only?) used for file access logs so might be worth moving here?

Reply:
Oh good catch, thanks.  Checked and they're all only used in access_log_manager_impl.* so I think moving them is the right thing to do.  Shame about the prefix